diff --git a/resources/views/category/partials/listing/item.blade.php b/resources/views/category/partials/listing/item.blade.php
index 0714391..d544e34 100644
--- a/resources/views/category/partials/listing/item.blade.php
+++ b/resources/views/category/partials/listing/item.blade.php
@@ -5,6 +5,7 @@
             <x-rapidez::no-image v-else class="rounded-t h-48 mb-3"/>
             <div class="px-2">
                 <div class="hyphens">@{{ item.name }}</div>
+                <stars class="my-1" :score="item.reviews_score" :count="item.reviews_count"></stars>
                 @if (!Rapidez::config('catalog/frontend/show_swatches_in_product_list', 1))
                     <div class="font-semibold">@{{ item.price | price}}</div>
                 @endif
diff --git a/resources/views/product/overview.blade.php b/resources/views/product/overview.blade.php
index 24d80f5..0827974 100644
--- a/resources/views/product/overview.blade.php
+++ b/resources/views/product/overview.blade.php
@@ -4,7 +4,10 @@
 @section('description', $product->meta_description)
 
 @section('content')
-    <h1 class="font-bold text-4xl mb-5">{{ $product->name }}</h1>
+    <div class="flex items-center mb-5">
+        <h1 class="font-bold text-4xl mr-3">{{ $product->name }}</h1>
+        <stars :score="{{ $product->reviews_score }}" :count="{{ $product->reviews_count }}"></stars>
+    </div>
 
     <div class="flex flex-col sm:flex-row mb-5">
         <div class="sm:w-2/3 sm:mr-5">
@@ -34,6 +37,9 @@
         @endforeach
     </dl>
 
+    @include('rapidez-reviews::reviews', ['sku' => $product->sku])
+    @include('rapidez-reviews::form', ['sku' => $product->sku])
+
     <x-rapidez::productlist title="Related products" field="id" :value="$product->relation_ids"/>
     <x-rapidez::productlist title="We found other products you might like!" field="id" :value="$product->upsell_ids"/>
 @endsection
